include/MFRC522_func.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * Helper routine to dump a byte array as hex values to Serial.
+ */
+void dump_byte_array(byte *buffer, byte bufferSize) {
+    for (byte i = 0; i < bufferSize; i++) {
+        Serial.print(buffer[i] < 0x10 ? " 0" : " ");
+        Serial.print(buffer[i], HEX);
+    }
+}
+
+bool authenticate(byte trailerBlock, MFRC522::MIFARE_Key key) {
+    MFRC522::StatusCode status;
+
+    // Authenticate using key A
+    // Serial.println(F("Authenticating using key A..."));
+    status = (MFRC522::StatusCode) mfrc522.PCD_Authenticate(MFRC522::PICC_CMD_MF_AUTH_KEY_A, trailerBlock, &key, &(mfrc522.uid));
```

Comment:
What is `mfrc522`? are you reading it from global scope?? 😱
Why not pass it as function argument instead?
```
bool authenticate(const MFRC522 &mfrc522, ...)
```